tests/utils/renumber_negative_ids.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+try:
+    from lxml import etree
+except ImportError:
+    import xml.etree.ElementTree as etree
+import sys
+
+if len(sys.argv) < 3:
```

Comment:
Optional: maybe delegate all checks and help strings to argparse?

Reply:
Okay.